budget/index.html

```diff
@@ -16,12 +16,27 @@
 {% block content_main %}
 
     <section class="media block block__flush-top block__sub">
-        <div class="media_image-container media_image-container__wide-margin media_image-container__right">
+        <div class="media_image-container
+                    media_image-container__wide-margin
+                    media_image-container__right">
             <img class="media_image" src="">
         </div>
         <div class="media_body">
             <h1>Budget and strategy</h1>
-            <p class="h3">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit.<p>
+            <p class="h3">The mission of the CFPB is to make consumer finance rules more
+            effective, to consistently and fairly enforce those rules, and to empower consumers
+            to take more control over their economic lives. Our vision is a consumer finance
+            marketplace:</p>
+            <ul class="h3 list list__branded">
+                <li>where customers can see prices and risks up front and where they can easily
+                make product comparisons;</li>
```

Comment:
Should this and the below list item end with a semicolon and the last one end with a period? This looks like a sentence that was converted to a bulleted list, but kept its formatting for a sentence.